app/src/main/java/com/android/feedme/ui/find/FindRecipeScreen.kt

```diff
@@ -98,17 +171,48 @@ fun FindRecipeScreen(navigationActions: NavigationActions, inputViewModel: Input
               )
 
               // Line separator
```

Comment:
Is this commented code gonna be removed ?

Reply:
Yes ! I removed that